src/adapters.ts

```diff
@@ -455,6 +453,27 @@ export function AdapterL1<TBase extends Constructor<TxSender>>(Base: TBase) {
       }
     }
 
+    async getNativeTokenVaultL1(): Promise<ethers.Contract> {
+      // FIXME: maybe makes sense to provide an API to do it in one call
+      const bridgeContracts =
+        await this._providerL2().getDefaultBridgeAddresses();
+
+      const sharedBridge = bridgeContracts.sharedL1;
+      const l1AR = IL1AssetRouter__factory.connect(
```

Comment:
nit: update `l1AR` to something more descriptive please. Suggestions:
- `l1AssetRouter`